packaging/update-wiki.sh

```diff
@@ -27,6 +36,8 @@ git config --local user.email "orabot@users.noreply.github.com"
 git config --local user.name "orabot"
 git add "Traits${TAG}.md"
 git add "Lua API${TAG}.md"
-git commit -m "Update trait and scripting documentation for branch $1"
+git commit -m "Update trait and scripting documentation for branch $1" &&
```

Comment:
`git commit` returns 1 when there was nothing to commit, in which case we can skip the push